protokt-codegen/src/main/kotlin/protokt/v1/codegen/Main.kt

```diff
@@ -63,14 +63,12 @@ private fun main(`in`: InputStream, out: OutputStream) {
 
     val grpcKotlinFiles = generateGrpcKotlinStubs(params, req)
 
-    if (files.isNotEmpty() || grpcKotlinFiles.isNotEmpty()) {
```

Comment:
If you generate nothing and the input has a file that uses proto3 `optional`, protoc fails since you didn't reply that you support the feature. We just didn't happen to trigger this. IIRC I added this check to handle an issue we had when trying to return no files, but clearly that's not a problem anymore.